src/stk/molecular/bonds/bond.py

```diff
@@ -172,6 +184,19 @@ def is_periodic(self):
 
         return any(direction != 0 for direction in self._periodicity)
 
+    def is_dative(self):
+        """
+        Return ``True`` if the bond is dative.
+
+        Returns
+        -------
+        :class:`bool`
+            ``True`` if the bond is dative.
+
+        """
+
+        return self._is_dative is True
```

Comment:
the `is True` here is redundant. Just `return self._is_dative` will give the same result.